.php_cs.dist

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/* TODO Refactor in src file
+if (!file_exists(__DIR__.'/src')) {
+    exit(0);
+}
+*/
+
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__)
+    ->exclude('vendor')
+    ->exclude('tests/tmp')
+;
+
+return PhpCsFixer\Config::create()
+    ->setRules([
+        '@Symfony' => true,
+        'declare_strict_types' => true,
+        'final_class' => true,
```

Comment:
```suggestion
       // 'final_class' => true,
```

Because this rule needs a review for every class that is used as base for others (extends)